test/test.js

```diff
@@ -10,6 +11,19 @@ const githubAPIMock = nock('https://api.github.com');
 
 chai.use(chaiHttp);
 
+describe('Github service', () => {
+    it('should use Authorization header and a token provided from config.js', async () => { 
+        githubAPIMock.matchHeader('Authorization', `Bearer ${config.githubToken}`).post('/graphql');
+        await github.searchRepositories('WordsMemorizer');
+
+        const isHeaderCorrect = githubAPIMock.isDone();
+
+        if(!isHeaderCorrect) githubAPIMock.cleanAll();
+        
+        return isHeaderCorrect;
```

Comment:
Is it mandatory to return any value here?

Reply:
It is not. :)